doc/user/tutorial.rst

```diff
@@ -466,6 +466,76 @@ Query Strings
 *Coming soon...*
 
 
+Cookies
```

Comment:
This looks more like reference material than something that would belong in the tutorial... Could you move this documentation to a new file, `doc/api/cookies.rst`? You can then refer to it from the relevant Request and Response docstrings.